### ISSUE TITLE
Pip says cykhash.unique, cykhash.sets, and cykhash.maps can't be installed as data

### ISSUE THREAD

**Issue by eafpres** [realead/cykhash#8]
Python 3.9
Unbuntu 20.04

```
eafpres@LAPTOP-JSLJ8BRD:~$ pip install https://github.com/realead/cykhash/zipball/master
Collecting https://github.com/realead/cykhash/zipball/master
  Downloading https://github.com/realead/cykhash/zipball/master
     / 527 kB 709 kB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Building wheels for collected packages: cykhash
  Building wheel for cykhash (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 /tmp/tmp9frys2yl_in_process.py build_wheel /tmp/tmpurpu24sq
       cwd: /tmp/pip-req-build-nefha6pc
  Complete output (135 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-cpython-39
  creating build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/__init__.py -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/compat.py -> build/lib.linux-x86_64-cpython-39/cykhash
  running egg_info
  writing src/cykhash.egg-info/PKG-INFO
  writing dependency_links to src/cykhash.egg-info/dependency_links.txt
  writing top-level names to src/cykhash.egg-info/top_level.txt
  reading manifest file 'src/cykhash.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE'
  writing manifest file 'src/cykhash.egg-info/SOURCES.txt'
  /tmp/pip-build-env-ek5jod3h/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'cykhash.maps' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'cykhash.maps' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'cykhash.maps' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'cykhash.maps' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.


  !!

    check.warn(importable)
  /tmp/pip-build-env-ek5jod3h/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'cykhash.sets' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'cykhash.sets' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'cykhash.sets' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'cykhash.sets' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.


  !!

    check.warn(importable)
  /tmp/pip-build-env-ek5jod3h/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'cykhash.unique' as data is deprecated, please list it in `packages`.
      !!


      ############################
      # Package would be ignored #
      ############################
      Python recognizes 'cykhash.unique' as an importable package,
      but it is not listed in the `packages` configuration of setuptools.

      'cykhash.unique' has been automatically added to the distribution only
      because it may contain data files, but this behavior is likely to change
      in future versions of setuptools (and therefore is considered deprecated).

      Please make sure that 'cykhash.unique' is included as a package by using
      the `packages` configuration field or the proper discovery methods
      (for example by using `find_namespace_packages(...)`/`find_namespace:`
      instead of `find_packages(...)`/`find:`).

      You can read more about "package discovery" and "data files" on setuptools
      documentation page.


  !!

    check.warn(importable)
  copying src/cykhash/common.pxi -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/floatdef.pxd -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/hash_functions.pxi -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/khash.pxi -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/khashmaps.pxd -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/khashmaps.pyx -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/khashsets.pxd -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/khashsets.pyx -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/memory.pxi -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/murmurhash.pxi -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/unique.pxd -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/unique.pyx -> build/lib.linux-x86_64-cpython-39/cykhash
  copying src/cykhash/utils.pyx -> build/lib.linux-x86_64-cpython-39/cykhash
  creating build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_header.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_header.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_impl.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_impl.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_init.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  copying src/cykhash/maps/map_init.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/maps
  creating build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_header.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_header.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_impl.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_impl.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_init.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  copying src/cykhash/sets/set_init.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/sets
  creating build/lib.linux-x86_64-cpython-39/cykhash/unique
  copying src/cykhash/unique/unique_impl.pxi -> build/lib.linux-x86_64-cpython-39/cykhash/unique
  copying src/cykhash/unique/unique_impl.pxi.in -> build/lib.linux-x86_64-cpython-39/cykhash/unique
  running build_ext
  building 'cykhash.khashsets' extension
  creating build/temp.linux-x86_64-cpython-39
  creating build/temp.linux-x86_64-cpython-39/src
  creating build/temp.linux-x86_64-cpython-39/src/cykhash
  x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/usr/include/python3.9 -c src/cykhash/khashsets.c -o build/temp.linux-x86_64-cpython-39/src/cykhash/khashsets.o
  src/cykhash/khashsets.c:19:10: fatal error: Python.h: No such file or directory
     19 | #include "Python.h"
        |          ^~~~~~~~~~
  compilation terminated.
  error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for cykhash
Failed to build cykhash
ERROR: Could not build wheels for cykhash which use PEP 517 and cannot be installed directly
```

**Comment by pxv8780** [comment]
Same issue,
Python 3.9
Mac M1, ARM64

**Comment by realead** [comment]
The error is due to python-dev not being installed, see https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory

I leave it open, because I should get read of the warning "Package would be ignored"

**Comment by realead** [comment]
The warning has been fixed with 2.0.1.

The package was not installed because there is no developer python was installed.
